specfile/rpm.py

```diff
@@ -284,9 +287,13 @@ def make_dummy_sources(sources: List[str], sourcedir: Path) -> Iterator[List[Pat
         for path in dummy_sources:
             path.unlink()
 
-    @staticmethod
+    @classmethod
     def parse(
-        content: str, sourcedir: Path, macros: Optional[List[Tuple[str, str]]] = None
+        cls,
+        content: str,
+        sourcedir: Path,
+        macros: Optional[List[Tuple[str, str]]] = None,
+        force: bool = False,
     ) -> rpm.spec:
```

Comment:
Shouldn't this be `Tuple[rpm.spec, bool]`? I find it a bit weird that mypy is silent about this.

Reply:
Thanks, yes, this was an oversight. I'm also wondering how come it passed mypy checks :thinking: